### PR TITLE
Migrate Version Column

### PR DESCRIPTION
**Pull request by deanmarcussen**
Fixes an Orchard Core Issue https://github.com/OrchardCMS/OrchardCore/issues/4585 introduced with https://github.com/sebastienros/yessql/pull/208

The migration that was there doesn't run if the table exists.

This seemed like the best way to fix it. 
Run a single query on the document table and look for the version column
You may have better ideas, so change or do differently if you want :)

Only tested under sqlite, but I think we got the TOP / LIMIT issues on MS SQL fixed another time.

And it seems the tests pass second time around

### CODE REVIEW

**Comment by deanmarcussen** [comment]
On reflection this is not the best way to fix this for perf

`SELECT TOP 1 Version` (with the right dialects) returned as a scalar will throw at the right times, and the code that was there to create the table or throw, and add the version column.

I won't be able to amend until tonight

File: src/YesSql.Core/Store.cs [hunk 1]
```diff
@@ -117,21 +117,53 @@ public async Task InitializeCollectionAsync(string collectionName)
...
+                                }
+                                return;
+                            }
                         }
                     }
                 }
-                catch
+                catch 
                 {
                     using (var transaction = connection.BeginTransaction())
```
Comment (sebastienros) [inline src/YesSql.Core/Store.cs]: Remove this inner catch

Comment (deanmarcussen) [reply src/YesSql.Core/Store.cs]: I think it needs it to handle the table create if the table itself doesn't exist?

Comment (sebastienros) [reply src/YesSql.Core/Store.cs]: right sorry, I thought it was the one I added, but you already removed it.

**Pull request merged**